packages/core/src/integrations/captureconsole.ts

```diff
@@ -63,10 +63,12 @@ function consoleHandler(args: unknown[], level: string): void {
       return event;
     });
 
-    if (level === 'assert' && args[0] === false) {
-      const message = `Assertion failed: ${safeJoin(args.slice(1), ' ') || 'console.assert'}`;
-      scope.setExtra('arguments', args.slice(1));
-      captureMessage(message, captureContext);
+    if (level === 'assert') {
+      if (args[0] === false) {
```

Comment:
This keeps the same comparison as before, but actually `console.assert` will fail for anything falsey. So maybe this should be `if (!args[0])`

Reply:
let's make this match what assert does 👍